plugins/foreman_openscap/0.5/index.md

```diff
@@ -108,7 +119,7 @@ Edit ```/etc/foreman-proxy/settings.d/openscap.yml``` with the appropriate setti
 
     puppet module install isimluk-foreman_scap_client
 
-This puppet module will automatically install foreman_scap_client (if not installed) and will configure the client's ```/etc/foreman_scap_client/config.yaml``` with parameters which are needed for the operation of foreman_scap_client.
+This puppet module will automatically install foreman_scap_client (if not installed) and will configure the client's ```/etc/foreman_scap_client/config.yaml``` with parameters which are needed for the operation of foreman_scap_client. The module is used from Foreman to automate the configuration and trigering of foreman_client_client, so you should install it in all puppet environments that your hosts use.
```

Comment:
"is used by Foreman" "triggering foreman_openscap_client" (I presume it's openscap_client?)